docs/data/sdks/flutter/index.md

```diff
@@ -59,7 +59,39 @@ class YourClass {
 }
 ```
 
-### EU data residency
+### Configuration
+
+Amplitude Flutter SDK runs on the top of the [Amplitude Android Mantaince SDK](../android/), [Amplitude iOS Mantaince SDK](../ios/) and [Amplitude JavaScript Mantaince SDK](../javascript/). The following are the Dart settable config options. For more configurations on Android side, please check the [Android Configuration](../android-kotlin/#configuration). For more configurations on iOS side, please check the [iOS configuration](../ios/#configuration). For more configurations on browser side, please check the [browser configuration](../javascript/#configuration).
```

Comment:
Might be better it we use bullet list here for better format and readability: 

```suggestion
Amplitude Flutter SDK runs on the top of the [Amplitude Android Mantaince SDK](../android/), [Amplitude iOS Mantaince SDK](../ios/) and [Amplitude JavaScript Mantaince SDK](../javascript/). The following are the Dart settable config options.

For more configurations:

- on Android side, check the [Android Configuration](../android-kotlin/#configuration)
- on iOS side, check the [iOS configuration](../ios/#configuration)
- on browser side, please check the [browser configuration](../javascript/#configuration)```

Reply:
Good Idea.